<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
    ~~~~~~~~~~~~~~~~^
        recipe,
        ^^^^^^^
        fix=fix,
        ^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
    ~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/check_syntax.py", line 36, in check_recipe
    has_constraints = constraints.search(spec)
TypeError: expected string or bytes-like object, got 'NoneType'

ERROR conda.cli.main_run:execute(127): `conda run conda-lint /tmp/abs_c5royt_oz4/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -26,7 +26,7 @@ requirements:
   build:
     - {{ stdlib("c") }}
     - {{ compiler("c") }}
-    - m2-patch  # [win]
+    - msys2-patch  # [win]
   host:
     - python
     - setuptools

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -33,9 +33,9 @@ requirements:
     - pip
     - wheel  
     - dsdp 5.8
-    - fftw 3.3.9
-    - glpk 4.65
-    - gsl 2.7.1
+    - fftw {{ fftw }}
+    - glpk {{ glpk }}
+    - gsl {{ gsl }}
     - openblas-devel {{ openblas_devel }}  # [blas_impl == "openblas"]
     # suitesparse is linked against on windows, and compiled-in on windows, because yeah, windows.
     - suitesparse {{ suitesparse }}  # [not win]

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -38,7 +38,7 @@ requirements:
     - gsl 2.7.1
     - openblas-devel {{ openblas_devel }}  # [blas_impl == "openblas"]
     # suitesparse is linked against on windows, and compiled-in on windows, because yeah, windows.
-    - suitesparse 7.8.3  # [not win]
+    - suitesparse {{ suitesparse }}  # [not win]
   run:
     - python
     - gsl

